File: 백준/Bronze/11531. ACM 대회 채점/ACM 대회 채점.c
#include <stdio.h>
#include <stdlib.h>

int main(void)
{
        int     n;
        char    c;
        char    s[6];
        int     arr[26];
        int     total;
        int     cnt;

        for (int i = 0; i <= 25; i++)
                arr[i] = 0;
        total = 0;
        cnt = 0;
        while (1)
        {
                scanf("%d", &n);
                if (n == -1)
                        break;
                scanf(" %c %s", &c, s);
                if (s[0] == 'w')
                        arr[c - 65] += 1;
                else
                {
                        total = total + n + arr[c - 65] * 20;
                        cnt += 1;
                }
        }
        printf("%d %d", cnt, total);
}

File: 백준/Bronze/11531. ACM 대회 채점/ACM 대회 채점.py

d_count = {}
total = 0
cnt = 0
while (1):
    arr = input().split()
    arr[0] = int(arr[0])

    if (arr[0] == -1):
        break

    if (arr[2] == 'right'):
        if (arr[1] in d_count):
            total += arr[0] + d_count[arr[1]] * 20
        else:
            total += arr[0]
        cnt += 1
    else:
        if (arr[1] in d_count):
            d_count[arr[1]] += 1
        else:
            d_count[arr[1]] = 1
print(cnt, total)